packages/core/src/checkout-buttons/checkout-button-options.ts

```diff
@@ -119,4 +107,22 @@ export interface CheckoutButtonInitializeOptions extends CheckoutButtonOptions {
      * They can be omitted unless you need to support Authorize.Net GooglePay.
      */
     googlepayauthorizenet?: GooglePayButtonInitializeOptions;
+
+    /**
+     * The options that are required to facilitate PayPal. They can be omitted
+     * unless you need to support Paypal.
+     */
+    paypal?: PaypalButtonInitializeOptions;
+
+    /**
+     * The options that are required to facilitate PayPal Commerce. They can be omitted
+     * unless you need to support Paypal.
+     */
+    paypalCommerce?: PaypalCommerceButtonInitializeOptions;
```

Comment:
```suggestion
    paypalcommerce?: PaypalCommerceButtonInitializeOptions;
```

Reply:
It will be changed in one of the future pr